code/datums/mind.dm

```diff
@@ -102,6 +102,7 @@
 		leave_all_huds() //leave all the huds in the old body, so it won't get huds if somebody else enters it
 
 		SSnanoui.user_transferred(current, new_character)
+		SStgui.on_transfer(current, new_character)
```

Comment:
Forgot to add this when I was doing the initial port, this updates UIs when you mindswap so youre using your new mob's UI state checks